From 8030f3f402903cc288312b401a2f33150d7be8df Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Wed, 27 Dec 2017 22:47:08 +0100
Subject: [PATCH 1/6] LineageParts: Deprecate few button settings

* Recents long press action hasn't been used for anything
  since CM11 days.
* Left side navigation bar isn't necessary as Oreo makes
  the navigation bar stick to the bottom of the phone.

Change-Id: I9ea0ca0ad7caad851508e144ab6fd4411dfb870c
---
 res/values-af/strings.xml                          |  2 -
 res/values-ast-rES/strings.xml                     |  2 -
 res/values-be-rBY/strings.xml                      |  1 -
 res/values-bg/strings.xml                          |  2 -
 res/values-ca/strings.xml                          |  2 -
 res/values-cs/strings.xml                          |  2 -
 res/values-da/strings.xml                          |  2 -
 res/values-de/strings.xml                          |  2 -
 res/values-el/strings.xml                          |  2 -
 res/values-en-rAU/strings.xml                      |  2 -
 res/values-en-rGB/strings.xml                      |  2 -
 res/values-en-rIN/strings.xml                      |  2 -
 res/values-es-rUS/strings.xml                      |  2 -
 res/values-es/strings.xml                          |  2 -
 res/values-eu-rES/strings.xml                      |  2 -
 res/values-fi/strings.xml                          |  2 -
 res/values-fr/strings.xml                          |  2 -
 res/values-gl-rES/strings.xml                      |  2 -
 res/values-hr/strings.xml                          |  2 -
 res/values-hu/strings.xml                          |  2 -
 res/values-in/strings.xml                          |  2 -
 res/values-it/strings.xml                          |  2 -
 res/values-iw/strings.xml                          |  2 -
 res/values-ja/strings.xml                          |  2 -
 res/values-ko/strings.xml                          |  2 -
 res/values-nb/strings.xml                          |  2 -
 res/values-nl/strings.xml                          |  2 -
 res/values-pl/strings.xml                          |  2 -
 res/values-pt-rBR/strings.xml                      |  2 -
 res/values-pt-rPT/strings.xml                      |  2 -
 res/values-ro/strings.xml                          |  2 -
 res/values-ru/strings.xml                          |  2 -
 res/values-sk/strings.xml                          |  2 -
 res/values-sl/strings.xml                          |  2 -
 res/values-sq-rAL/strings.xml                      |  2 -
 res/values-sr/strings.xml                          |  2 -
 res/values-tr/strings.xml                          |  2 -
 res/values-uk/strings.xml                          |  2 -
 res/values-vi/strings.xml                          |  2 -
 res/values-zh-rCN/strings.xml                      |  2 -
 res/values-zh-rTW/strings.xml                      |  2 -
 res/values/strings.xml                             |  2 -
 res/xml/button_settings.xml                        | 12 ---
 .../lineageparts/input/ButtonSettings.java         | 94 ----------------------
 44 files changed, 189 deletions(-)

diff --git a/res/values-af/strings.xml b/res/values-af/strings.xml
index 2a6b1e0..8de2699 100644
--- a/res/values-af/strings.xml
+++ b/res/values-af/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigasiebalk</string>
   <string name="navigation_bar_title">Knoppies en uitleg</string>
-  <string name="navigation_bar_left_title">Linkshandige modus</string>
   <string name="navigation_bar_left_summary">Plaas die navigasie balk aan die linkerkant van die skerm in landskap modus</string>
   <string name="navigation_bar_arrow_keys_title">Wys pyltjiesleutels terwyl jy tik</string>
   <string name="navigation_bar_arrow_keys_summary">Wys links en regs wyser knoppies terwyl tik. Oorskryf IME wisselaar.</string>
   <string name="navigation_bar_home_long_press_title">Tuis lang druk aksie</string>
   <string name="navigation_bar_home_double_tap_title">Tuis dubbel tik aksie</string>
-  <string name="navigation_bar_recents_title">Onlangse lang-drukaksie</string>
   <!-- Power menu -->
   <string name="power_menu_title">Krag kieslys</string>
   <string name="power_menu_reboot_title">Herlaai kieslys</string>
diff --git a/res/values-ast-rES/strings.xml b/res/values-ast-rES/strings.xml
index 8778cd9..c3bb991 100644
--- a/res/values-ast-rES/strings.xml
+++ b/res/values-ast-rES/strings.xml
@@ -156,9 +156,7 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Barra de navegación</string>
   <string name="navigation_bar_title">Botones y distribución</string>
-  <string name="navigation_bar_left_title">Mou pa maniegos</string>
   <string name="navigation_bar_left_summary">Alluga la barra de navegación nel llau esquierdu de la pantalla nel mou apaisáu</string>
-  <string name="navigation_bar_recents_title">Aición de primíu llargu de Recientes</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menú d\'apagáu</string>
   <string name="power_menu_reboot_title">Menú de reaniciu</string>
diff --git a/res/values-be-rBY/strings.xml b/res/values-be-rBY/strings.xml
index c8a9006..d9f6886 100644
--- a/res/values-be-rBY/strings.xml
+++ b/res/values-be-rBY/strings.xml
@@ -166,7 +166,6 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Навігацыйны радок</string>
   <string name="navigation_bar_title">Кнопкі і іх месцазнаходжаньне</string>
-  <string name="navigation_bar_left_title">Ляўшун</string>
   <string name="navigation_bar_left_summary">Адлюстроўваць навігацыйны радок зьлева ў альбомнай арыентацыі</string>
   <string name="navigation_bar_arrow_keys_title">Паказваць стрэлкі падчас друкаваньня тэксту</string>
   <string name="navigation_bar_arrow_keys_summary">Паказваць кнопки левага и правага курсору падчас друкаваньня. Кнопка варыянту друкаваньня будзе схована.</string>
diff --git a/res/values-bg/strings.xml b/res/values-bg/strings.xml
index dc90ba1..61b7af7 100644
--- a/res/values-bg/strings.xml
+++ b/res/values-bg/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Навигационна лента</string>
   <string name="navigation_bar_title">Бутони и оформление</string>
-  <string name="navigation_bar_left_title">Режим на лява ръка</string>
   <string name="navigation_bar_left_summary">Поставете лентата за навигация от лявата страна на екрана в пейзажен режим</string>
   <string name="navigation_bar_arrow_keys_title">Показване на стрелките при въвеждане</string>
   <string name="navigation_bar_arrow_keys_summary">Показване на ляв и десен курсорен бутони при въвеждане. Замества IME превключвател.</string>
   <string name="navigation_bar_home_long_press_title">Действие при дълго натисканане на начало</string>
   <string name="navigation_bar_home_double_tap_title">Действие при двукратно натискане на начало</string>
-  <string name="navigation_bar_recents_title">Последни дълги натискания действие</string>
   <!-- Power menu -->
   <string name="power_menu_title">Меню мощност</string>
   <string name="power_menu_reboot_title">Меню рестартиране</string>
diff --git a/res/values-ca/strings.xml b/res/values-ca/strings.xml
index 4855958..57f7d8f 100644
--- a/res/values-ca/strings.xml
+++ b/res/values-ca/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Barra de navegació</string>
   <string name="navigation_bar_title">Botons i disposició</string>
-  <string name="navigation_bar_left_title">Mode esquerrà</string>
   <string name="navigation_bar_left_summary">Col·loca la barra de navegació al costat esquerre de la pantalla en el mode horitzontal</string>
   <string name="navigation_bar_arrow_keys_title">Mostra tecles de les fletxes mentre escrius</string>
   <string name="navigation_bar_arrow_keys_summary">Mostra els botons dels cursors esquerra i dreta mentre escrius. Això anul·la el commutador IME.</string>
   <string name="navigation_bar_home_long_press_title">Acció de pulsació llarga d\'Inici</string>
   <string name="navigation_bar_home_double_tap_title">Acció de doble pulsació d\'Inici</string>
-  <string name="navigation_bar_recents_title">Acció pulsació llarga de Recents</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menú d\'apagat</string>
   <string name="power_menu_reboot_title">Menú de reinici</string>
diff --git a/res/values-cs/strings.xml b/res/values-cs/strings.xml
index 475dde4..58949b8 100644
--- a/res/values-cs/strings.xml
+++ b/res/values-cs/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigační lišta</string>
   <string name="navigation_bar_title">Tlačítka a rozmístění</string>
-  <string name="navigation_bar_left_title">Režim ovládání levou rukou</string>
   <string name="navigation_bar_left_summary">Při natočení na šířku umístit navigační lištu na levé straně obrazovky</string>
   <string name="navigation_bar_arrow_keys_title">Zobrazit navigační šipky při psaní</string>
   <string name="navigation_bar_arrow_keys_summary">Zobrazit levý a pravý kurzor při psaní. Potlačí IME přepínač.</string>
   <string name="navigation_bar_home_long_press_title">Akce dlouhého stisku tlačítka „Domů“</string>
   <string name="navigation_bar_home_double_tap_title">Akce dvojitého stisku tlačítka „Domů“</string>
-  <string name="navigation_bar_recents_title">Akce dlouhého stisku tlačítka „Nedávné“</string>
   <!-- Power menu -->
   <string name="power_menu_title">Nabídka tlačítka Vypnout</string>
   <string name="power_menu_reboot_title">Nabídka restartu</string>
diff --git a/res/values-da/strings.xml b/res/values-da/strings.xml
index 1b9225b..dde83be 100644
--- a/res/values-da/strings.xml
+++ b/res/values-da/strings.xml
@@ -211,13 +211,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigationslinje</string>
   <string name="navigation_bar_title">Knapper og layout</string>
-  <string name="navigation_bar_left_title">Venstrehåndstilstand</string>
   <string name="navigation_bar_left_summary">Anbring navigeringslinjen på venstre side af skærmen i landskabstilstand</string>
   <string name="navigation_bar_arrow_keys_title">Vis piletaster, mens du skriver</string>
   <string name="navigation_bar_arrow_keys_summary">Vis venstre og højre piletaster, når du skriver. Tilsidesætter IME-indstillinger.</string>
   <string name="navigation_bar_home_long_press_title">Langt tryk-handling på knappen Hjem</string>
   <string name="navigation_bar_home_double_tap_title">Handling ved dobbelt tryk på knappen Hjem</string>
-  <string name="navigation_bar_recents_title">Langt tryk-handling på knappen Seneste handlinger</string>
   <!-- Power menu -->
   <string name="power_menu_title">Tænd/sluk-menu</string>
   <string name="power_menu_reboot_title">Genstartsmenu</string>
diff --git a/res/values-de/strings.xml b/res/values-de/strings.xml
index d48eec5..ff18355 100644
--- a/res/values-de/strings.xml
+++ b/res/values-de/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigationsleiste</string>
   <string name="navigation_bar_title">Schaltflächen und Anordnung</string>
-  <string name="navigation_bar_left_title">Linkshänder-Modus</string>
   <string name="navigation_bar_left_summary">Navigationsleiste im Querformat auf der linken Seite anzeigen</string>
   <string name="navigation_bar_arrow_keys_title">Pfeiltasten beim Tippen anzeigen</string>
   <string name="navigation_bar_arrow_keys_summary">Beim Tippen Links/Rechts-Cursortasten anzeigen (deaktiviert die Eingabemethodenauswahl).</string>
   <string name="navigation_bar_home_long_press_title">Home (langes Drücken)</string>
   <string name="navigation_bar_home_double_tap_title">Home (doppeltes Tippen)</string>
-  <string name="navigation_bar_recents_title">Langes Drücken auf Anwendungsverlauf</string>
   <!-- Power menu -->
   <string name="power_menu_title">Ein-/Aus-Menü</string>
   <string name="power_menu_reboot_title">Neustartmenü</string>
diff --git a/res/values-el/strings.xml b/res/values-el/strings.xml
index e9d70ed..2e1158b 100644
--- a/res/values-el/strings.xml
+++ b/res/values-el/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Γραμμή πλοήγησης</string>
   <string name="navigation_bar_title">Πλήκτρα και διάταξη</string>
-  <string name="navigation_bar_left_title">Λειτουργία για αριστερόχειρες</string>
   <string name="navigation_bar_left_summary">Τοποθέτηση της γραμμής πλοήγησης στην αριστερή πλευρά της οθόνης κατά την οριζόντια λειτουργία</string>
   <string name="navigation_bar_arrow_keys_title">Εμφάνιση βελών κατά την πληκτρολόγηση</string>
   <string name="navigation_bar_arrow_keys_summary">Εμφάνιση πλήκτρων αριστερού και δεξιού κέρσορα κατά την πληκτρολόγηση. Παρακάμπτει το εικονίδιο επιλογέα.</string>
   <string name="navigation_bar_home_long_press_title">Ενέργεια παρατεταμένης πίεσης αρχικής οθόνης</string>
   <string name="navigation_bar_home_double_tap_title">Ενέργεια διπλού χτυπήματος αρχικής οθόνης</string>
-  <string name="navigation_bar_recents_title">Ενέργεια παρατεταμένης πίεσης προσφάτων</string>
   <!-- Power menu -->
   <string name="power_menu_title">Μενού πλήκτρου απενεργοποίησης</string>
   <string name="power_menu_reboot_title">Μενού επανεκκίνησης</string>
diff --git a/res/values-en-rAU/strings.xml b/res/values-en-rAU/strings.xml
index c66b07b..4c91b2f 100644
--- a/res/values-en-rAU/strings.xml
+++ b/res/values-en-rAU/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigation bar</string>
   <string name="navigation_bar_title">Buttons and layout</string>
-  <string name="navigation_bar_left_title">Left-handed mode</string>
   <string name="navigation_bar_left_summary">Place the navigation bar on the left side of the screen in landscape mode</string>
   <string name="navigation_bar_arrow_keys_title">Show arrow keys while typing</string>
   <string name="navigation_bar_arrow_keys_summary">Display left and right cursor buttons while typing. Overrides IME switcher.</string>
   <string name="navigation_bar_home_long_press_title">Home long press action</string>
   <string name="navigation_bar_home_double_tap_title">Home double tap action</string>
-  <string name="navigation_bar_recents_title">Recents long press action</string>
   <!-- Power menu -->
   <string name="power_menu_title">Power menu</string>
   <string name="power_menu_reboot_title">Restart menu</string>
diff --git a/res/values-en-rGB/strings.xml b/res/values-en-rGB/strings.xml
index f73b4c2..695af32 100644
--- a/res/values-en-rGB/strings.xml
+++ b/res/values-en-rGB/strings.xml
@@ -211,13 +211,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigation bar</string>
   <string name="navigation_bar_title">Buttons and layout</string>
-  <string name="navigation_bar_left_title">Left-handed mode</string>
   <string name="navigation_bar_left_summary">Place the navigation bar on the left side of the screen in landscape mode</string>
   <string name="navigation_bar_arrow_keys_title">Show arrow keys while typing</string>
   <string name="navigation_bar_arrow_keys_summary">Display left and right cursor buttons while typing. Overrides IME switcher.</string>
   <string name="navigation_bar_home_long_press_title">Home long press action</string>
   <string name="navigation_bar_home_double_tap_title">Home double tap action</string>
-  <string name="navigation_bar_recents_title">Recents long press action</string>
   <!-- Power menu -->
   <string name="power_menu_title">Power menu</string>
   <string name="power_menu_reboot_title">Restart menu</string>
diff --git a/res/values-en-rIN/strings.xml b/res/values-en-rIN/strings.xml
index 064dce6..a9dfc4d 100644
--- a/res/values-en-rIN/strings.xml
+++ b/res/values-en-rIN/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigation bar</string>
   <string name="navigation_bar_title">Buttons and layout</string>
-  <string name="navigation_bar_left_title">Left-handed mode</string>
   <string name="navigation_bar_left_summary">Place the navigation bar on the left side of the screen in landscape mode</string>
   <string name="navigation_bar_arrow_keys_title">Show arrow keys while typing</string>
   <string name="navigation_bar_arrow_keys_summary">Display left and right cursor buttons while typing. Overrides IME switcher.</string>
   <string name="navigation_bar_home_long_press_title">Home long press action</string>
   <string name="navigation_bar_home_double_tap_title">Home double tap action</string>
-  <string name="navigation_bar_recents_title">Recents long press action</string>
   <!-- Power menu -->
   <string name="power_menu_title">Power menu</string>
   <string name="power_menu_reboot_title">Restart menu</string>
diff --git a/res/values-es-rUS/strings.xml b/res/values-es-rUS/strings.xml
index e0bb1cc..69081ca 100644
--- a/res/values-es-rUS/strings.xml
+++ b/res/values-es-rUS/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Barra de navegación</string>
   <string name="navigation_bar_title">Botones y disposición</string>
-  <string name="navigation_bar_left_title">Modo para zurdos</string>
   <string name="navigation_bar_left_summary">Colocar la barra de navegación a la izquierda de la pantalla en modo horizontal</string>
   <string name="navigation_bar_arrow_keys_title">Mostrar teclas de flechas al escribir</string>
   <string name="navigation_bar_arrow_keys_summary">Mostrar el botón izquierdo y derecho del cursor al escribir. Reemplaza al selector IME.</string>
   <string name="navigation_bar_home_long_press_title">Acción de pulsación larga en Inicio</string>
   <string name="navigation_bar_home_double_tap_title">Acción de doble pulsación en Inicio</string>
-  <string name="navigation_bar_recents_title">Acción de pulsación larga en recientes</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menú de apagado</string>
   <string name="power_menu_reboot_title">Menú de reinicio</string>
diff --git a/res/values-es/strings.xml b/res/values-es/strings.xml
index c14ebd7..32a1c5c 100644
--- a/res/values-es/strings.xml
+++ b/res/values-es/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Barra de navegación</string>
   <string name="navigation_bar_title">Botones y disposición</string>
-  <string name="navigation_bar_left_title">Modo para zurdos</string>
   <string name="navigation_bar_left_summary">Situar la barra de navegación a la izquierda de la pantalla en el modo horizontal</string>
   <string name="navigation_bar_arrow_keys_title">Mostrar flechas al escribir</string>
   <string name="navigation_bar_arrow_keys_summary">Mostrar los cursores izquierda y derecha durante la escritura. Modifica al selector IME.</string>
   <string name="navigation_bar_home_long_press_title">Acción de pulsación larga en inicio</string>
   <string name="navigation_bar_home_double_tap_title">Acción de doble toque en inicio</string>
-  <string name="navigation_bar_recents_title">Acción de pulsación larga en recientes</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menú de apagado</string>
   <string name="power_menu_reboot_title">Menú de reinicio</string>
diff --git a/res/values-eu-rES/strings.xml b/res/values-eu-rES/strings.xml
index f4938db..69212c9 100644
--- a/res/values-eu-rES/strings.xml
+++ b/res/values-eu-rES/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Nabigazio-barra</string>
   <string name="navigation_bar_title">Botoiak eta diseinua</string>
-  <string name="navigation_bar_left_title">Ezkertientzako modua</string>
   <string name="navigation_bar_left_summary">Kokatu nabigazio-barra pantailaren ezkerraldean paisaia moduan</string>
   <string name="navigation_bar_arrow_keys_title">Erakutsi norabide geziak idaztean</string>
   <string name="navigation_bar_arrow_keys_summary">Erakutsi ezker zein eskuin kurtsore botoiak idaztean. IME aldatzailea gainidazten du.</string>
   <string name="navigation_bar_home_long_press_title">Hasiera botoian luze zapaltzearen ekintza</string>
   <string name="navigation_bar_home_double_tap_title">Hasiera botoian birritan sakatzearen ekintza</string>
-  <string name="navigation_bar_recents_title">Azkenak botoian luze sakatzearen ekintza</string>
   <!-- Power menu -->
   <string name="power_menu_title">Itzaltze menua</string>
   <string name="power_menu_reboot_title">Berrabiarazte menua</string>
diff --git a/res/values-fi/strings.xml b/res/values-fi/strings.xml
index ea16927..e06c78e 100644
--- a/res/values-fi/strings.xml
+++ b/res/values-fi/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigointipalkki</string>
   <string name="navigation_bar_title">Näppäimet ja sijainti</string>
-  <string name="navigation_bar_left_title">Käytä vasemmalla kädellä</string>
   <string name="navigation_bar_left_summary">Siirtää navigointipalkin näytön vasempaan reunaan vaaka-asennossa</string>
   <string name="navigation_bar_arrow_keys_title">Näytä nuolinäppäimet kirjottaessa</string>
   <string name="navigation_bar_arrow_keys_summary">Näytä vasen ja oikea nuolinäppäin kirjoittaessa. Ohittaa syöttötavan valinnan.</string>
   <string name="navigation_bar_home_long_press_title">Kotinäppäimen pitkän painalluksen tehtävä</string>
   <string name="navigation_bar_home_double_tap_title">Kotinäppäimen kaksoisnapautuksen tehtävä</string>
-  <string name="navigation_bar_recents_title">Viimeisimmät-painikkeen pitkän painalluksen tehtävä</string>
   <!-- Power menu -->
   <string name="power_menu_title">Käynnistysvalikko</string>
   <string name="power_menu_reboot_title">Uudelleenkäynnistysvalikko</string>
diff --git a/res/values-fr/strings.xml b/res/values-fr/strings.xml
index d596b0a..8c471b1 100644
--- a/res/values-fr/strings.xml
+++ b/res/values-fr/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Barre de navigation</string>
   <string name="navigation_bar_title">Boutons et disposition</string>
-  <string name="navigation_bar_left_title">Mode gaucher</string>
   <string name="navigation_bar_left_summary">Placer la barre de navigation du côté gauche de l\'écran en mode paysage</string>
   <string name="navigation_bar_arrow_keys_title">Afficher les flèches lors de la frappe</string>
   <string name="navigation_bar_arrow_keys_summary">Afficher les touches de curseur gauche et droite lors de la saisie. Remplace le changeur de mode de saisie.</string>
   <string name="navigation_bar_home_long_press_title">Appui long sur bouton d\'accueil</string>
   <string name="navigation_bar_home_double_tap_title">Appui double sur bouton d\'accueil</string>
-  <string name="navigation_bar_recents_title">Appui long sur applications récentes</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menu marche/arrêt</string>
   <string name="power_menu_reboot_title">Menu redémarrer</string>
diff --git a/res/values-gl-rES/strings.xml b/res/values-gl-rES/strings.xml
index d624a01..b2c991b 100644
--- a/res/values-gl-rES/strings.xml
+++ b/res/values-gl-rES/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Barra de navegación</string>
   <string name="navigation_bar_title">Botóns e disposición</string>
-  <string name="navigation_bar_left_title">Modo para zurdo</string>
   <string name="navigation_bar_left_summary">Colocar a barra de navegación no lado esquerdo da pantalla cando estea en modo horizontal</string>
   <string name="navigation_bar_arrow_keys_title">Amosar as frechas ao escribir</string>
   <string name="navigation_bar_arrow_keys_summary">Amosar os cursores esquerda e dereita ao escribir. Substitúe ao selector IME.</string>
   <string name="navigation_bar_home_long_press_title">Acción de pulsación larga do inicio</string>
   <string name="navigation_bar_home_double_tap_title">Acción con toque duplo en inicio</string>
-  <string name="navigation_bar_recents_title">Toque un tempo en aplicacións recentes</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menú de apagado</string>
   <string name="power_menu_reboot_title">Menú de reinicio</string>
diff --git a/res/values-hr/strings.xml b/res/values-hr/strings.xml
index 12aff74..c782cc7 100644
--- a/res/values-hr/strings.xml
+++ b/res/values-hr/strings.xml
@@ -204,13 +204,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigacijska traka</string>
   <string name="navigation_bar_title">Tipke i izgled</string>
-  <string name="navigation_bar_left_title">Način rada za ljevake</string>
   <string name="navigation_bar_left_summary">Postavlja navigacijsku traku na lijevu stranu zaslona u vodoravnom načinu rada</string>
   <string name="navigation_bar_arrow_keys_title">Prikaži strelice tijekom upisivanja</string>
   <string name="navigation_bar_arrow_keys_summary">Prikazuje lijeve i desne kursorske tipke tijekom unosa teksta. Zamjenjuje tipku za promjenu tipkovnice.</string>
   <string name="navigation_bar_home_long_press_title">Akcija dugog dodira Home tipke</string>
   <string name="navigation_bar_home_double_tap_title">Dvostruki dodir</string>
-  <string name="navigation_bar_recents_title">Nedavne akcije dugog dodira</string>
   <!-- Power menu -->
   <string name="power_menu_title">Izbornik napajanja</string>
   <string name="power_menu_reboot_title">Izbornik Ponovnog pokretanja</string>
diff --git a/res/values-hu/strings.xml b/res/values-hu/strings.xml
index 39afce0..3532bd0 100644
--- a/res/values-hu/strings.xml
+++ b/res/values-hu/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigációs sáv</string>
   <string name="navigation_bar_title">Gombok és kinézet</string>
-  <string name="navigation_bar_left_title">Bal kezes mód</string>
   <string name="navigation_bar_left_summary">Fekvő módban a navigációs sáv megjelenítése bal oldalon</string>
   <string name="navigation_bar_arrow_keys_title">Nyílbillentyűk megjelenítése gépelés közben</string>
   <string name="navigation_bar_arrow_keys_summary">Mutassa a bal és jobb kurzort gépelés közben. Felülírja a billentyűzet választó.</string>
   <string name="navigation_bar_home_long_press_title">Kezdőlap hosszan nyomási művelet</string>
   <string name="navigation_bar_home_double_tap_title">Kezdőlap dupla érintési művelet</string>
-  <string name="navigation_bar_recents_title">Hosszan nyomva tartott előzmények gomb</string>
   <!-- Power menu -->
   <string name="power_menu_title">Kikapcsoló menü</string>
   <string name="power_menu_reboot_title">Újraindítás menü</string>
diff --git a/res/values-in/strings.xml b/res/values-in/strings.xml
index 2a45f71..27e801e 100644
--- a/res/values-in/strings.xml
+++ b/res/values-in/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Bar Navigasi</string>
   <string name="navigation_bar_title">Tombol dan tata letak</string>
-  <string name="navigation_bar_left_title">Mode tangan kiri</string>
   <string name="navigation_bar_left_summary">Tempatkan navigasi bar di sebelah kiri layar dalam mode lanskap</string>
   <string name="navigation_bar_arrow_keys_title">Tampilkan tombol panah saat mengetik</string>
   <string name="navigation_bar_arrow_keys_summary">Tampilkan tombol kursor kiri dan kanan saat mengetik. Menggantikan pengubah IME.</string>
   <string name="navigation_bar_home_long_press_title">Tindakan menekan lama home</string>
   <string name="navigation_bar_home_double_tap_title">Tindakan dua kali ketuk home</string>
-  <string name="navigation_bar_recents_title">Tindakan menekan lama recent</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menu power</string>
   <string name="power_menu_reboot_title">Mulai ulang menu</string>
diff --git a/res/values-it/strings.xml b/res/values-it/strings.xml
index b804140..ea356d5 100644
--- a/res/values-it/strings.xml
+++ b/res/values-it/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Barra di navigazione</string>
   <string name="navigation_bar_title">Pulsanti e disposizione</string>
-  <string name="navigation_bar_left_title">Modalità mancino</string>
   <string name="navigation_bar_left_summary">Posiziona la barra di navigazione a sinistra quando il dispositivo è ruotato orizzontalmente</string>
   <string name="navigation_bar_arrow_keys_title">Mostra frecce mentre scrivi</string>
   <string name="navigation_bar_arrow_keys_summary">Mostra i pulsanti sinistra e destra del cursore mentre stai scrivendo. Nasconde il selettore di immissione.</string>
   <string name="navigation_bar_home_long_press_title">Azione pressione prolungata Home</string>
   <string name="navigation_bar_home_double_tap_title">Azione doppio tap Home</string>
-  <string name="navigation_bar_recents_title">Azione pressione prolungata Panoramica</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menu di accensione</string>
   <string name="power_menu_reboot_title">Menu riavvio</string>
diff --git a/res/values-iw/strings.xml b/res/values-iw/strings.xml
index b4c8d75..2f3c513 100644
--- a/res/values-iw/strings.xml
+++ b/res/values-iw/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">סרגל ניווט</string>
   <string name="navigation_bar_title">מקשים ופריסה</string>
-  <string name="navigation_bar_left_title">מצב לשמאליים</string>
   <string name="navigation_bar_left_summary">כאשר המסך מאוזן מקם את סרגל הניווט בשמאל המסך</string>
   <string name="navigation_bar_arrow_keys_title">הראה חיצים בעת הקלדה</string>
   <string name="navigation_bar_arrow_keys_summary">הצג מקשי ימין ושמאל בעת הכתיבה. במקום מחליף המקלדות.</string>
   <string name="navigation_bar_home_long_press_title">פעולת לחיצה ארוכה על מקש הבית</string>
   <string name="navigation_bar_home_double_tap_title">פעולת לחיצה כפולה על מקש הבית</string>
-  <string name="navigation_bar_recents_title">פעולת לחיצה ארוכה על מקש יישומים אחרונים</string>
   <!-- Power menu -->
   <string name="power_menu_title">תפריט כיבוי</string>
   <string name="power_menu_reboot_title">תפריט הפעלה מחדש</string>
diff --git a/res/values-ja/strings.xml b/res/values-ja/strings.xml
index 8d03ade..2178e69 100644
--- a/res/values-ja/strings.xml
+++ b/res/values-ja/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">ナビゲーションバー</string>
   <string name="navigation_bar_title">ボタンとレイアウト</string>
-  <string name="navigation_bar_left_title">左利きモード</string>
   <string name="navigation_bar_left_summary">横画面モードでナビゲーションバーを画面の左側に配置する</string>
   <string name="navigation_bar_arrow_keys_title">入力中は矢印キーを表示</string>
   <string name="navigation_bar_arrow_keys_summary">入力中は左右の矢印ボタンを表示します。IMEの切り替えアイコンは無効になります。</string>
   <string name="navigation_bar_home_long_press_title">ホームボタン長押しの動作</string>
   <string name="navigation_bar_home_double_tap_title">ホームボタンのダブルタップの動作</string>
-  <string name="navigation_bar_recents_title">履歴ボタンの長押しの動作</string>
   <!-- Power menu -->
   <string name="power_menu_title">電源メニュー</string>
   <string name="power_menu_reboot_title">再起動メニュー</string>
diff --git a/res/values-ko/strings.xml b/res/values-ko/strings.xml
index 57d4d55..84eaf21 100644
--- a/res/values-ko/strings.xml
+++ b/res/values-ko/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">내비게이션 바</string>
   <string name="navigation_bar_title">버튼 및 레이아웃</string>
-  <string name="navigation_bar_left_title">왼손잡이 모드</string>
   <string name="navigation_bar_left_summary">화면 가로 모드에서 화면의 왼쪽에 내비게이션 바를 배치</string>
   <string name="navigation_bar_arrow_keys_title">입력 도중 화살표 키 표시</string>
   <string name="navigation_bar_arrow_keys_summary">입력 시 좌우 커서 버튼을 표시합니다. 입력기 전환기보다 우선합니다.</string>
   <string name="navigation_bar_home_long_press_title">홈 버튼을 길게 누를 시 동작 설정</string>
   <string name="navigation_bar_home_double_tap_title">홈 버튼을 두 번 누를 시 동작 설정</string>
-  <string name="navigation_bar_recents_title">멀티태스킹 버튼을 길게 누를 시 동작 설정</string>
   <!-- Power menu -->
   <string name="power_menu_title">전원 메뉴</string>
   <string name="power_menu_reboot_title">다시 시작 메뉴</string>
diff --git a/res/values-nb/strings.xml b/res/values-nb/strings.xml
index 7a35696..0737949 100644
--- a/res/values-nb/strings.xml
+++ b/res/values-nb/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigasjonsfelt</string>
   <string name="navigation_bar_title">Knapper og oppsett</string>
-  <string name="navigation_bar_left_title">Venstrehendt modus</string>
   <string name="navigation_bar_left_summary">Plasser navigasjonsfeltet til venstre på skjermen i liggende modus</string>
   <string name="navigation_bar_arrow_keys_title">Vis piltastene mens du skriver</string>
   <string name="navigation_bar_arrow_keys_summary">Vise venstre og høyre markørknapper mens du skriver. Overstyrer IME skifter.</string>
   <string name="navigation_bar_home_long_press_title">Hjem langt trykk handling</string>
   <string name="navigation_bar_home_double_tap_title">Hjem dobbelttrykk handling</string>
-  <string name="navigation_bar_recents_title">Siste langt trykk handling</string>
   <!-- Power menu -->
   <string name="power_menu_title">Av/På meny</string>
   <string name="power_menu_reboot_title">Omstartsmeny</string>
diff --git a/res/values-nl/strings.xml b/res/values-nl/strings.xml
index d988e44..3674c24 100644
--- a/res/values-nl/strings.xml
+++ b/res/values-nl/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigatiebalk</string>
   <string name="navigation_bar_title">Knoppen en indeling</string>
-  <string name="navigation_bar_left_title">Linkshandige modus</string>
   <string name="navigation_bar_left_summary">Navigatiebalk aan de linkerkant van het scherm weergeven in liggende stand</string>
   <string name="navigation_bar_arrow_keys_title">Pijltjestoetsen tijdens typen</string>
   <string name="navigation_bar_arrow_keys_summary">Cursor links en rechts weergeven tijdens typen. Verbergt invoermethodepictogram.</string>
   <string name="navigation_bar_home_long_press_title">Thuisknop lang indrukken</string>
   <string name="navigation_bar_home_double_tap_title">Thuisknop twee keer indrukken</string>
-  <string name="navigation_bar_recents_title">Recente apps-knop lang indrukken</string>
   <!-- Power menu -->
   <string name="power_menu_title">Opties in afsluitmenu</string>
   <string name="power_menu_reboot_title">Herstarten</string>
diff --git a/res/values-pl/strings.xml b/res/values-pl/strings.xml
index c8cef6b..fd75c21 100644
--- a/res/values-pl/strings.xml
+++ b/res/values-pl/strings.xml
@@ -212,13 +212,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Pasek nawigacji</string>
   <string name="navigation_bar_title">Przyciski i ich układ</string>
-  <string name="navigation_bar_left_title">Tryb leworęczny</string>
   <string name="navigation_bar_left_summary">W trybie poziomym pokaż pasek nawigacji po lewej stronie ekranu</string>
   <string name="navigation_bar_arrow_keys_title">Pokaż klawisze strzałek podczas pisania</string>
   <string name="navigation_bar_arrow_keys_summary">Pokaż przyciski lewego i prawego kursora podczas pisania. Zastępuje przełącznik IME.</string>
   <string name="navigation_bar_home_long_press_title">Akcja długiego naciśnięcia Home</string>
   <string name="navigation_bar_home_double_tap_title">Akcja podwójnego puknięcia Home</string>
-  <string name="navigation_bar_recents_title">Akcja długiego naciśnięcia Ostatnio Używanych</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menu zasilania</string>
   <string name="power_menu_reboot_title">Menu ponownego uruchomienia</string>
diff --git a/res/values-pt-rBR/strings.xml b/res/values-pt-rBR/strings.xml
index 46cc96a..08ab046 100644
--- a/res/values-pt-rBR/strings.xml
+++ b/res/values-pt-rBR/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Barra de navegação</string>
   <string name="navigation_bar_title">Botões e layout</string>
-  <string name="navigation_bar_left_title">Modo canhoto</string>
   <string name="navigation_bar_left_summary">Posicionar a barra de navegação no lado esquerdo da tela em modo paisagem</string>
   <string name="navigation_bar_arrow_keys_title">Mostrar teclas de seta enquanto digita</string>
   <string name="navigation_bar_arrow_keys_summary">Exibir botões esquerda e direita do cursor durante a digitação. Substitui o seletor de Teclado.</string>
   <string name="navigation_bar_home_long_press_title">Ação de toque longo no botão início</string>
   <string name="navigation_bar_home_double_tap_title">Ação de toque duplo no botão de Página Inicial</string>
-  <string name="navigation_bar_recents_title">Ação de pressionar e segurar aplicativos recentes</string>
   <!-- Power menu -->
   <string name="power_menu_title">Opções do botão liga/desliga</string>
   <string name="power_menu_reboot_title">Menu de reinicialização</string>
diff --git a/res/values-pt-rPT/strings.xml b/res/values-pt-rPT/strings.xml
index 43d3c00..1433961 100644
--- a/res/values-pt-rPT/strings.xml
+++ b/res/values-pt-rPT/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Barra de navegação</string>
   <string name="navigation_bar_title">Botões e sua disposição</string>
-  <string name="navigation_bar_left_title">Modo de mão esquerda</string>
   <string name="navigation_bar_left_summary">Coloca a barra de navegação no lado esquerdo do ecrã, quando estiver no modo paisagem</string>
   <string name="navigation_bar_arrow_keys_title">Mostrar teclas de seta enquanto escreve</string>
   <string name="navigation_bar_arrow_keys_summary">Mostrar os botões de cursor para a esquerda e para a direita enquanto escreve. Sobrepõe-se à definição do teclado.</string>
   <string name="navigation_bar_home_long_press_title">Ação de pressão longa no botão de Página Inicial</string>
   <string name="navigation_bar_home_double_tap_title">Toque duplo</string>
-  <string name="navigation_bar_recents_title">Ação de pressão longa no botão de Visão geral</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menu ligar/desligar</string>
   <string name="power_menu_reboot_title">Menu de reinicialização</string>
diff --git a/res/values-ro/strings.xml b/res/values-ro/strings.xml
index ee53dd1..8b49f70 100644
--- a/res/values-ro/strings.xml
+++ b/res/values-ro/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Bară de navigare</string>
   <string name="navigation_bar_title">Butoane și aspect</string>
-  <string name="navigation_bar_left_title">Mod stângaci</string>
   <string name="navigation_bar_left_summary">Pune bara de navigare în partea stângă a ecranului în modul peisaj</string>
   <string name="navigation_bar_arrow_keys_title">Arată butoanele de direcție în timp ce tastați</string>
   <string name="navigation_bar_arrow_keys_summary">Arată butoanele de cursor stânga şi dreapta în timp ce tastaţi. Înlocuieşte comutatorul IME.</string>
   <string name="navigation_bar_home_long_press_title">Acțiunea apăsării lungi pe Acasă</string>
   <string name="navigation_bar_home_double_tap_title">Acțiunea apăsării duble pe Acasă</string>
-  <string name="navigation_bar_recents_title">Acțiunea apăsării lungi Recente</string>
   <!-- Power menu -->
   <string name="power_menu_title">Meniul power</string>
   <string name="power_menu_reboot_title">Meniu repornire</string>
diff --git a/res/values-ru/strings.xml b/res/values-ru/strings.xml
index fe63cc2..51258dc 100644
--- a/res/values-ru/strings.xml
+++ b/res/values-ru/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Панель навигации</string>
   <string name="navigation_bar_title">Кнопки и их расположение</string>
-  <string name="navigation_bar_left_title">Режим левши</string>
   <string name="navigation_bar_left_summary">Отображать панель навигации слева в альбомной ориентации экрана</string>
   <string name="navigation_bar_arrow_keys_title">Показать стрелки при вводе текста</string>
   <string name="navigation_bar_arrow_keys_summary">Отображать кнопки перемещения курсора влево или вправо при вводе текста. Кнопка выбора метода ввода отображаться не будет.</string>
   <string name="navigation_bar_home_long_press_title">Долгое нажатие кнопки «Домой»</string>
   <string name="navigation_bar_home_double_tap_title">Двойное нажатие кнопки «Домой»</string>
-  <string name="navigation_bar_recents_title">Долгое нажатие кнопки «Обзор»</string>
   <!-- Power menu -->
   <string name="power_menu_title">Меню кнопки питания</string>
   <string name="power_menu_reboot_title">Меню перезапуска</string>
diff --git a/res/values-sk/strings.xml b/res/values-sk/strings.xml
index 197aae2..5600eaa 100644
--- a/res/values-sk/strings.xml
+++ b/res/values-sk/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Navigačný panel</string>
   <string name="navigation_bar_title">Tlačidlá a rozloženie</string>
-  <string name="navigation_bar_left_title">Režim pre ľavákov</string>
   <string name="navigation_bar_left_summary">V režime na šírku umiestniť navigačný panel na ľavú stranu obrazovky</string>
   <string name="navigation_bar_arrow_keys_title">Zobraziť šípky pri písaní</string>
   <string name="navigation_bar_arrow_keys_summary">Zobraziť ľavé a pravé kurzorové tlačidlá pri písaní. Potlačí prepínač metódy vstupu.</string>
   <string name="navigation_bar_home_long_press_title">Akcia dlhého stlačenia tlačidla Domov</string>
   <string name="navigation_bar_home_double_tap_title">Akcia dvojkliku tlačidla Domov</string>
-  <string name="navigation_bar_recents_title">Akcia dlhého stlačenia tlačidla Nedávne</string>
   <!-- Power menu -->
   <string name="power_menu_title">Ponuka po stlačení tlačidla</string>
   <string name="power_menu_reboot_title">Reštart menu</string>
diff --git a/res/values-sl/strings.xml b/res/values-sl/strings.xml
index 5dc2445..21ca0c7 100644
--- a/res/values-sl/strings.xml
+++ b/res/values-sl/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Vrstica za krmarjenje</string>
   <string name="navigation_bar_title">Gumbi in razporeditev</string>
-  <string name="navigation_bar_left_title">Način za levičarje</string>
   <string name="navigation_bar_left_summary">Umesti vrstico za krmarjenje ne levo stran zaslona v ležečem načinu</string>
   <string name="navigation_bar_arrow_keys_title">Prikaži smerne tipke med tipkanjem</string>
   <string name="navigation_bar_arrow_keys_summary">Prikaži gumba levega in desnega kazalca med tipkanjem. Preglasi preklopnik IME.</string>
   <string name="navigation_bar_home_long_press_title">Dejanje dolgega pritiska Domov</string>
   <string name="navigation_bar_home_double_tap_title">Dejanje dvojnega tapa Domov</string>
-  <string name="navigation_bar_recents_title">Dejanje dolgega pritiska Nedavno</string>
   <!-- Power menu -->
   <string name="power_menu_title">Meni gumba za vklop/izklop</string>
   <string name="power_menu_reboot_title">Meni ponovnega zagona</string>
diff --git a/res/values-sq-rAL/strings.xml b/res/values-sq-rAL/strings.xml
index 37207fc..334be36 100644
--- a/res/values-sq-rAL/strings.xml
+++ b/res/values-sq-rAL/strings.xml
@@ -211,13 +211,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Shiriti i lundrimit</string>
   <string name="navigation_bar_title">Butonat dhe pozicionimi</string>
-  <string name="navigation_bar_left_title">Përdorimi me dorën e majtë</string>
   <string name="navigation_bar_left_summary">Vendos shiritin e lundrimit në anën e majtë të ekranit kur ai është i kthyer horizontalisht</string>
   <string name="navigation_bar_arrow_keys_title">Trego butonat e shigjetave gjatë kohës që shkruan</string>
   <string name="navigation_bar_arrow_keys_summary">Shfaq butonat e kursorit të majtë dhe të djathtë gjatë kohës që shkruan. Ky opsion shpërfill çelësin IME.</string>
   <string name="navigation_bar_home_long_press_title">Veprimi kur mban shtypur butonin Home</string>
   <string name="navigation_bar_home_double_tap_title">Shtypja dy herë e butonit Home</string>
-  <string name="navigation_bar_recents_title">Veprimi kur mban shtypur butonin Të fundit</string>
   <!-- Power menu -->
   <string name="power_menu_title">Menuja e energjisë</string>
   <string name="power_menu_reboot_title">Menuja e rindezjes</string>
diff --git a/res/values-sr/strings.xml b/res/values-sr/strings.xml
index 5ed3776..318f32a 100644
--- a/res/values-sr/strings.xml
+++ b/res/values-sr/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Навигациона трака</string>
   <string name="navigation_bar_title">Дугмад и распоред</string>
-  <string name="navigation_bar_left_title">Режим за леворуке</string>
   <string name="navigation_bar_left_summary">Поставља навигациону траку слева у водоравном режиму.</string>
   <string name="navigation_bar_arrow_keys_title">Стрелице при уносу текста</string>
   <string name="navigation_bar_arrow_keys_summary">Приказује дугмад за померање курсора улево или удесно при уносу текста. Дугме за промену метода уноса неће бити приказано.</string>
   <string name="navigation_bar_home_long_press_title">Дуги притисак на дугме „Почетна“</string>
   <string name="navigation_bar_home_double_tap_title">Двоструки додир дугмета „Почетна“</string>
-  <string name="navigation_bar_recents_title">Дуги притисак на дугме „Преглед“</string>
   <!-- Power menu -->
   <string name="power_menu_title">Мени за напајање</string>
   <string name="power_menu_reboot_title">Мени за поновно покретање</string>
diff --git a/res/values-tr/strings.xml b/res/values-tr/strings.xml
index 818ae3c..992011c 100644
--- a/res/values-tr/strings.xml
+++ b/res/values-tr/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Gezinme çubuğu</string>
   <string name="navigation_bar_title">Tuşlar ve düzen</string>
-  <string name="navigation_bar_left_title">Sol el ile kullanım modu</string>
   <string name="navigation_bar_left_summary">Yatay modda ekranın sol tarafına gezinme çubuğunu yerleştirir</string>
   <string name="navigation_bar_arrow_keys_title">Yazı yazarken ok tuşlarını göster</string>
   <string name="navigation_bar_arrow_keys_summary">Yazı yazarken sağ ve sol imleç tuşlarını göster. Klavye değiştiriciyi gösterme.</string>
   <string name="navigation_bar_home_long_press_title">Ana ekran tuşuna uzun basma eylemi</string>
   <string name="navigation_bar_home_double_tap_title">Ana ekran tuşuna çift dokunma eylemi</string>
-  <string name="navigation_bar_recents_title">Son uygulamalar tuşuna uzun basma eylemi</string>
   <!-- Power menu -->
   <string name="power_menu_title">Güç menüsü</string>
   <string name="power_menu_reboot_title">Yeniden başlat menüsü</string>
diff --git a/res/values-uk/strings.xml b/res/values-uk/strings.xml
index f354a4f..46569dc 100644
--- a/res/values-uk/strings.xml
+++ b/res/values-uk/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Панель навігації</string>
   <string name="navigation_bar_title">Кнопки та їх розміщення</string>
-  <string name="navigation_bar_left_title">Режим для лівші</string>
   <string name="navigation_bar_left_summary">Показувати панель навігації зліва в альбомній орієнтації</string>
   <string name="navigation_bar_arrow_keys_title">Показувати стрілки під час введення тексту</string>
   <string name="navigation_bar_arrow_keys_summary">Показувати кнопки переміщення курсору вліво або вправо при введенні. Кнопка вибору методу введення не буде відображатися.</string>
   <string name="navigation_bar_home_long_press_title">Дія при довгому натисканні кнопки \"Додому\"</string>
   <string name="navigation_bar_home_double_tap_title">Дія при подвійному натисканні на кнопку \"Додому\"</string>
-  <string name="navigation_bar_recents_title">Дія при довгому натисканні кнопки \"Нещодавні додатки\"</string>
   <!-- Power menu -->
   <string name="power_menu_title">Меню кнопки живлення</string>
   <string name="power_menu_reboot_title">Меню перезавантаження</string>
diff --git a/res/values-vi/strings.xml b/res/values-vi/strings.xml
index d84fccb..6d078c3 100644
--- a/res/values-vi/strings.xml
+++ b/res/values-vi/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">Thanh điều hướng</string>
   <string name="navigation_bar_title">Phím và cách bố trí</string>
-  <string name="navigation_bar_left_title">Chế độ thuận tay trái</string>
   <string name="navigation_bar_left_summary">Đặt thanh điều hướng bên trái màn hình ở chế độ ngang</string>
   <string name="navigation_bar_arrow_keys_title">Hiển thị phím mũi tên trong khi nhập liệu</string>
   <string name="navigation_bar_arrow_keys_summary">Hiển thị các nút con trỏ bên trái và bên phải khi nhập liệu. Ghi đè bộ chuyển IME.</string>
   <string name="navigation_bar_home_long_press_title">Hành động khi nhấn giữ phím Trang chính</string>
   <string name="navigation_bar_home_double_tap_title">Hành động nhấn phím Trang chính hai lần</string>
-  <string name="navigation_bar_recents_title">Hành động nhấn giữ phím Gần đây</string>
   <!-- Power menu -->
   <string name="power_menu_title">Trình đơn nguồn</string>
   <string name="power_menu_reboot_title">Trình đơn khởi động lại</string>
diff --git a/res/values-zh-rCN/strings.xml b/res/values-zh-rCN/strings.xml
index 4d20126..b8770ca 100644
--- a/res/values-zh-rCN/strings.xml
+++ b/res/values-zh-rCN/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">导航栏</string>
   <string name="navigation_bar_title">按键与布局</string>
-  <string name="navigation_bar_left_title">左利手模式</string>
   <string name="navigation_bar_left_summary">横屏时在屏幕左侧显示导航栏</string>
   <string name="navigation_bar_arrow_keys_title">键入时显示方向键</string>
   <string name="navigation_bar_arrow_keys_summary">键入时显示左右光标按钮而不显示输入法选择器。</string>
   <string name="navigation_bar_home_long_press_title">Home 键的长按操作</string>
   <string name="navigation_bar_home_double_tap_title">Home 键双击行为</string>
-  <string name="navigation_bar_recents_title">最近应用键的长按操作</string>
   <!-- Power menu -->
   <string name="power_menu_title">电源菜单</string>
   <string name="power_menu_reboot_title">重启菜单</string>
diff --git a/res/values-zh-rTW/strings.xml b/res/values-zh-rTW/strings.xml
index 3d0ab71..ca6b59a 100644
--- a/res/values-zh-rTW/strings.xml
+++ b/res/values-zh-rTW/strings.xml
@@ -213,13 +213,11 @@
   <!-- Navigation Bar -->
   <string name="navigation_bar_category">導覽列</string>
   <string name="navigation_bar_title">按鍵與配置</string>
-  <string name="navigation_bar_left_title">左撇子模式</string>
   <string name="navigation_bar_left_summary">橫向模式下將導覽列置於螢幕左側</string>
   <string name="navigation_bar_arrow_keys_title">輸入時顯示方向鍵</string>
   <string name="navigation_bar_arrow_keys_summary">輸入時顯示游標鍵。取代輸入法切換器。</string>
   <string name="navigation_bar_home_long_press_title">主畫面鍵長按動作</string>
   <string name="navigation_bar_home_double_tap_title">主畫面輕觸兩下動作</string>
-  <string name="navigation_bar_recents_title">多工鍵長按動作</string>
   <!-- Power menu -->
   <string name="power_menu_title">電源選單</string>
   <string name="power_menu_reboot_title">重新啟動選單</string>
diff --git a/res/values/strings.xml b/res/values/strings.xml
index cf589f0..7d19582 100644
--- a/res/values/strings.xml
+++ b/res/values/strings.xml
@@ -234,13 +234,11 @@
     <!-- Navigation Bar -->
     <string name="navigation_bar_category">Navigation bar</string>
     <string name="navigation_bar_title">Buttons and layout</string>
-    <string name="navigation_bar_left_title">Left-handed mode</string>
     <string name="navigation_bar_left_summary">Place the navigation bar on the left side of the screen in landscape mode</string>
     <string name="navigation_bar_arrow_keys_title">Show arrow keys while typing</string>
     <string name="navigation_bar_arrow_keys_summary">Display left and right cursor buttons while typing. Overrides IME switcher.</string>
     <string name="navigation_bar_home_long_press_title">Home long press action</string>
     <string name="navigation_bar_home_double_tap_title">Home double tap action</string>
-    <string name="navigation_bar_recents_title">Recents long press action</string>
 
     <!-- Power menu -->
     <string name="power_menu_title">Power menu</string>
diff --git a/res/xml/button_settings.xml b/res/xml/button_settings.xml
index 2ff6989..37661bb 100644
--- a/res/xml/button_settings.xml
+++ b/res/xml/button_settings.xml
@@ -38,12 +38,6 @@
         android:title="@string/navigation_bar_category" >
 
         <lineageos.preference.LineageSystemSettingSwitchPreference
-            android:key="navigation_bar_left"
-            android:title="@string/navigation_bar_left_title"
-            android:summary="@string/navigation_bar_left_summary"
-            android:defaultValue="false" />
-
-        <lineageos.preference.LineageSystemSettingSwitchPreference
                 android:key="navigation_bar_menu_arrow_keys"
                 android:title="@string/navigation_bar_arrow_keys_title"
                 android:summary="@string/navigation_bar_arrow_keys_summary"
@@ -71,12 +65,6 @@
             android:entryValues="@array/hardware_keys_action_values"
             android:persistent="false" />
 
-        <ListPreference
-            android:key="navigation_recents_long_press"
-            android:dialogTitle="@string/navigation_bar_recents_title"
-            android:title="@string/navigation_bar_recents_title"
-            android:persistent="false" />
-
     </PreferenceCategory>
 
     <PreferenceCategory
diff --git a/src/org/lineageos/lineageparts/input/ButtonSettings.java b/src/org/lineageos/lineageparts/input/ButtonSettings.java
index 92ef78e..3256733 100644
--- a/src/org/lineageos/lineageparts/input/ButtonSettings.java
+++ b/src/org/lineageos/lineageparts/input/ButtonSettings.java
@@ -69,10 +69,8 @@ public class ButtonSettings extends SettingsPreferenceFragment implements
     private static final String KEY_VOLUME_KEY_CURSOR_CONTROL = "volume_key_cursor_control";
     private static final String KEY_SWAP_VOLUME_BUTTONS = "swap_volume_buttons";
     private static final String DISABLE_NAV_KEYS = "disable_nav_keys";
-    private static final String KEY_NAVIGATION_BAR_LEFT = "navigation_bar_left";
     private static final String KEY_NAVIGATION_HOME_LONG_PRESS = "navigation_home_long_press";
     private static final String KEY_NAVIGATION_HOME_DOUBLE_TAP = "navigation_home_double_tap";
-    private static final String KEY_NAVIGATION_RECENTS_LONG_PRESS = "navigation_recents_long_press";
     private static final String KEY_POWER_END_CALL = "power_end_call";
     private static final String KEY_HOME_ANSWER_CALL = "home_answer_call";
     private static final String KEY_VOLUME_MUSIC_CONTROLS = "volbtn_music_controls";
@@ -148,10 +146,8 @@ public class ButtonSettings extends SettingsPreferenceFragment implements
     private SwitchPreference mVolumeMusicControls;
     private SwitchPreference mSwapVolumeButtons;
     private SwitchPreference mDisableNavigationKeys;
-    private SwitchPreference mNavigationBarLeftPref;
     private ListPreference mNavigationHomeLongPressAction;
     private ListPreference mNavigationHomeDoubleTapAction;
-    private ListPreference mNavigationRecentsLongPressAction;
     private SwitchPreference mPowerEndCall;
     private SwitchPreference mHomeAnswerCall;
     private SwitchPreference mTorchLongPressPowerGesture;
@@ -232,9 +228,6 @@ public class ButtonSettings extends SettingsPreferenceFragment implements
 
         mNavigationPreferencesCat = (PreferenceCategory) findPreference(CATEGORY_NAVBAR);
 
-        // Navigation bar left
-        mNavigationBarLeftPref = (SwitchPreference) findPreference(KEY_NAVIGATION_BAR_LEFT);
-
         Action defaultHomeLongPressAction = Action.fromIntSafe(res.getInteger(
                 com.android.internal.R.integer.config_longPressOnHomeBehavior));
         Action defaultHomeDoubleTapAction = Action.fromIntSafe(res.getInteger(
@@ -261,10 +254,6 @@ public class ButtonSettings extends SettingsPreferenceFragment implements
                 mNavigationPreferencesCat.removePreference(mNavigationHomeDoubleTapAction);
         }
 
-        // Navigation bar recents long press activity needs custom setup
-        mNavigationRecentsLongPressAction =
-                initRecentsLongPressAction(KEY_NAVIGATION_RECENTS_LONG_PRESS);
-
         final LineageHardwareManager hardware = LineageHardwareManager.getInstance(getActivity());
 
         // Only visible on devices that does not have a navigation bar already,
@@ -436,10 +425,6 @@ public class ButtonSettings extends SettingsPreferenceFragment implements
             boolean hasNavBar = WindowManagerGlobal.getWindowManagerService().hasNavigationBar()
                     || forceNavbar;
 
-            if (!ScreenType.isPhone(getActivity())) {
-                mNavigationPreferencesCat.removePreference(mNavigationBarLeftPref);
-            }
-
             if (!hasNavBar && (needsNavigationBar ||
                     !hardware.isSupported(LineageHardwareManager.FEATURE_KEY_DISABLE))) {
                     // Hide navigation bar category
@@ -510,71 +495,6 @@ public class ButtonSettings extends SettingsPreferenceFragment implements
         return list;
     }
 
-    private ListPreference initRecentsLongPressAction(String key) {
-        ListPreference list = (ListPreference) getPreferenceScreen().findPreference(key);
-        list.setOnPreferenceChangeListener(this);
-
-        // Read the componentName from Settings.Secure, this is the user's prefered setting
-        String componentString = LineageSettings.Secure.getString(getContentResolver(),
-                LineageSettings.Secure.RECENTS_LONG_PRESS_ACTIVITY);
-        ComponentName targetComponent = null;
-        if (componentString == null) {
-            list.setSummary(getString(R.string.hardware_keys_action_split_screen));
-        } else {
-            targetComponent = ComponentName.unflattenFromString(componentString);
-        }
-
-        // Dyanamically generate the list array,
-        // query PackageManager for all Activites that are registered for ACTION_RECENTS_LONG_PRESS
-        PackageManager pm = getPackageManager();
-        Intent intent = new Intent(lineageos.content.Intent.ACTION_RECENTS_LONG_PRESS);
-        List<ResolveInfo> recentsActivities = pm.queryIntentActivities(intent,
-                PackageManager.MATCH_DEFAULT_ONLY);
-        if (recentsActivities.size() == 0) {
-            // No entries available, disable
-            list.setSummary(getString(R.string.hardware_keys_action_split_screen));
-            LineageSettings.Secure.putString(getContentResolver(),
-                    LineageSettings.Secure.RECENTS_LONG_PRESS_ACTIVITY, null);
-            list.setEnabled(false);
-            return list;
-        }
-
-        CharSequence[] entries = new CharSequence[recentsActivities.size() + 1];
-        CharSequence[] values = new CharSequence[recentsActivities.size() + 1];
-        // First entry is always default last app
-        entries[0] = getString(R.string.hardware_keys_action_split_screen);
-        values[0] = "";
-        list.setValue(values[0].toString());
-        int i = 1;
-        for (ResolveInfo info : recentsActivities) {
-            try {
-                // Use pm.getApplicationInfo for the label,
-                // we cannot rely on ResolveInfo that comes back from queryIntentActivities.
-                entries[i] = pm.getApplicationInfo(info.activityInfo.packageName, 0).loadLabel(pm);
-            } catch (PackageManager.NameNotFoundException e) {
-                Log.e(TAG, "Error package not found: " + info.activityInfo.packageName, e);
-                // Fallback to package name
-                entries[i] = info.activityInfo.packageName;
-            }
-
-            // Set the value to the ComponentName that will handle this intent
-            ComponentName entryComponent = new ComponentName(info.activityInfo.packageName,
-                    info.activityInfo.name);
-            values[i] = entryComponent.flattenToString();
-            if (targetComponent != null) {
-                if (entryComponent.equals(targetComponent)) {
-                    // Update the selected value and the preference summary
-                    list.setSummary(entries[i]);
-                    list.setValue(values[i].toString());
-                }
-            }
-            i++;
-        }
-        list.setEntries(entries);
-        list.setEntryValues(values);
-        return list;
-    }
-
     private void handleListChange(ListPreference pref, Object newValue, String setting) {
         String value = (String) newValue;
         int index = pref.findIndexOfValue(value);
@@ -629,20 +549,6 @@ public class ButtonSettings extends SettingsPreferenceFragment implements
             handleSystemListChange(mVolumeKeyCursorControl, newValue,
                     Settings.System.VOLUME_KEY_CURSOR_CONTROL);
             return true;
-        } else if (preference == mNavigationRecentsLongPressAction) {
-            // RecentsLongPressAction is handled differently because it intentionally uses
-            // Settings.System
-            String putString = (String) newValue;
-            int index = mNavigationRecentsLongPressAction.findIndexOfValue(putString);
-            CharSequence summary = mNavigationRecentsLongPressAction.getEntries()[index];
-            // Update the summary
-            mNavigationRecentsLongPressAction.setSummary(summary);
-            if (putString.length() == 0) {
-                putString = null;
-            }
-            LineageSettings.Secure.putString(getContentResolver(),
-                    LineageSettings.Secure.RECENTS_LONG_PRESS_ACTIVITY, putString);
-            return true;
         } else if (preference == mTorchLongPressPowerTimeout) {
             handleListChange(mTorchLongPressPowerTimeout, newValue,
                     LineageSettings.System.TORCH_LONG_PRESS_POWER_TIMEOUT);
-- 
2.7.4

